packages/flutter/lib/src/rendering/list_wheel_viewport.dart

```diff
@@ -110,6 +113,7 @@ class RenderListWheelViewport
        assert(perspective != null),
        assert(perspective > 0),
        assert(perspective <= 0.01, perspectiveTooHighMessage),
+       assert(magnificationRate == null || magnificationRate > 0),
```

Comment:
Let them be separate asserts (like the other ones) so it's super clear to developers what went wrong when it breaks (the error message shows the exactly assert clause).

Reply:
fixed.